library_test.py

```diff
@@ -114,15 +114,46 @@ async def main() -> None:
 
     print("-" * 20)
     devices = await api.get_devices_data()
-    print("Devices:", devices)
+    print(f"Devices: {devices}")
+
     print("-" * 20)
     data = await api.get_sensor_data()
-    print("Data:", data)
```

Comment:
The aim of the `library_test` script is to check that data retrieval works, not to be used as stand-alone application.
So for me it doesn't make sense to format output.